bokehjs/src/lib/models/widgets/autocomplete_input.ts

```diff
@@ -133,6 +133,10 @@ export class AutocompleteInputView extends TextInputView {
       default: {
         const value = this.input_el.value
 
+        if (!this.model.strict) {
+          this.model.value = this.input_el.value
```

Comment:
This could use the const defined right above and be `this.model.value = value`

However, I'm slightly concerned and at least want to quickly discuss that these semantics are somewhat different to the non-strict case. When `Strict=True` it requires and `<enter>` or click to trigger `change_input` to actually update the model value. But it looks like when `strict=False` that the model state will be updated on every keystroke. I worry that this is both unexpected and surprising, and also problematic in that it does not afford the user any way to know when the user is "finished" typing

Reply:
You are certainly correct that the value is currently updated on every keystroke.
I thought that was also the behavior of a normal `TextInput` but as you mention that is not the case.
I agree that the behaviour should be essentially that of `TextInput` if completions are not used and I'll look into changing it accordingly.

Reply:
You might have a look at the parent class (in case you have not already) there is a `value_input` that *should` update every keystroke, and then `value` which should only update on a "confirm"

Reply:
It seems to me the simplest solution is to have `change_input` call `super.change_input()` as well.

If the change was made via the dropdown menu, the super call should do nothing. If not, it will ensure the same behavior as in the parent class.